crates/goose-server/src/routes/agent.rs

```diff
@@ -342,6 +357,146 @@ async fn update_session_config(
     }
 }
 
+#[utoipa::path(
```

Comment:
is utopia new or we use it elsewhere? (just not familiar with it)

Reply:
looks like it got [added](https://github.com/block/goose/commit/d8d78396e0cc9b0ec0049e98e630ecc66cf5df34#diff-f8b78f59b067067ff9bfef42f2cebe2f35c85bb230a4e735f56c8882cd5efc44R58) back in May (I think)